tools/amd_build/build_amd.py

```diff
@@ -63,6 +63,7 @@
     "caffe2/utils/*",
     "c10/cuda/*",
     "c10/cuda/test/CMakeLists.txt",
+    "modules/detectron/*",
```

Comment:
We can enable the entire "modules" subdirectory here, atm only detectron module has gpu files and when people add new gpu files we want to automatically enable in rocm.